### PR TITLE
[OBSDEF-8811] Create/Update KAHM events for licensing

### PR DESCRIPTION
**Pull request by SarahAFochs**
## Purpose
[OBSDEF-8811](https://jira.cec.lab.emc.com:8443/browse/OBSDEF-8811)
Additional rights added to rbac in order to issue events for licensing.

## PR checklist
- [x] make test passed
- [ ] make build passed
- [ ] helm install <chart> passed
- [ ] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed
- [x] Passed - https://asd-ecs-jenkins.isus.emc.com/jenkins/view/ECS-Flex/job/charts-custom-ci/

## Testing
_Paste URL of charts-custom-ci job here_

_Paste the output of your helm install/vSphere7 deployment testing here_

### CODE REVIEW

File: objectscale-manager/alerts/issues/OBJSC-LIC-0004_licensing_issue.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+- description: "ObjectScale cumulative object store usage"
+  name: "ObjectScale Licensing"
```
Comment (vgupta-mickey) [inline objectscale-manager/alerts/issues/OBJSC-LIC-0004_licensing_issue.yaml:2]: This issue should be auto clear by timeout

Comment (eddiepavkovic) [reply objectscale-manager/alerts/issues/OBJSC-LIC-0004_licensing_issue.yaml:2]: We send a Normal event after 6 hours if they're below 90% capacity.

**Review by eddiepavkovic — APPROVED** [review]
lgmt

File: objectscale-manager/alerts/issues/OBJSC-LIC-0004_licensing_issue.yaml (hunk 1 above)
Comment (vgupta-mickey) [reply objectscale-manager/alerts/issues/OBJSC-LIC-0004_licensing_issue.yaml:2]: Should we send any event at all if the thresold level is below the capacity? There will be so many normal events be accumulated and the issue will remain on the UI forever in Normal state and will not purge.

**Review by youminhan — APPROVED** [review]

**Pull request merged**